tests/blueprints/test_plates.py

```diff
@@ -4,7 +4,7 @@
 import responses  # type: ignore
 
 
-def test_post_plates_endpoint_successful(app, client, samples, mocked_responses):
+def test_post_plates_endpoint_successful(app, client, samples, mocked_responses, mlwh_lh_samples):
```

Comment:
Hmm, having trouble getting the mocking to work! But not sure it's necessary, because for the response code to be 'OK', it has to have called that and not thrown an exception?